app/models/subnet.rb

```diff
@@ -187,7 +187,14 @@ def self.import(proxy)
     ProxyAPI::DHCP.new(:url => proxy.url).subnets.map do |s|
       # do not import existing networks.
       attrs = { :network => s["network"], :mask => s["netmask"] }
+
       next if first(:conditions => attrs)
+      if s["options"]
+        attrs.merge!({:gateway => s["options"]["gateway"],
```

Comment:
Having said that, on the smart-proxy side I suggested to keep option names and format of values similar to the original ones. This would require explicit handling of each option (renaming, extracting of values, etc) here.